### PR TITLE
feat: add fs config to exclude mount binds

### PR DESCRIPTION
**Pull request by troian**

### CODE REVIEW

File: fs.go [hunk 1]
```diff
@@ -29,13 +30,19 @@ func (ca *Cagent) FSWatcher() *FSWatcher {
...
 		ExcludedPathCache: map[string]bool{},
 		cagent:            ca,
 	}
+
 	for _, t := range ca.Config.FSTypeInclude {
 		ca.fsWatcher.AllowedTypes[strings.ToLower(t)] = struct{}{}
 	}
 
+	for _, t := range ca.Config.FSPathExclude {
+		ca.fsWatcher.ExcludePath[strings.ToLower(t)] = struct{}{}
```
Comment (martinlindhe) [inline fs.go]: I think path should not be lowercased here. Case matters on Linux

File: fs.go [hunk 2]
```diff
@@ -49,25 +56,42 @@ func (fw *FSWatcher) Results() (common.MeasurementsMap, error) {
...
+				}
+			}
+		}
+
+		if pathExcluded {
+			continue
+		}
+
+		cacheExists := false
+		if pathExcluded, cacheExists = fw.ExcludedPathCache[strings.ToLower(partition.Mountpoint)]; cacheExists {
```
Comment (martinlindhe) [inline fs.go]: Again, not sure about lowercasing paths

File: fs.go (hunk 1 above)
Comment (troian) [reply fs.go]: done

File: fs.go (hunk 2 above)
Comment (troian) [reply fs.go]: done

File: fs.go [hunk 3]
```diff
@@ -36,11 +36,11 @@ func (ca *Cagent) FSWatcher() *FSWatcher {
 	}
 
 	for _, t := range ca.Config.FSTypeInclude {
-		ca.fsWatcher.AllowedTypes[strings.ToLower(t)] = struct{}{}
+		ca.fsWatcher.AllowedTypes[t] = struct{}{}
```
Comment (martinlindhe) [inline fs.go]: This part was from earlier code I think, and looked correct. The filesystem types should be expressed in lower case (ext3, not Ext3 or EXT3)

File: fs.go [hunk 4]
```diff
@@ -61,7 +61,7 @@ func (fw *FSWatcher) Results() (common.MeasurementsMap, error) {
 	}
 
 	for _, partition := range partitions {
-		if _, typeAllowed := fw.AllowedTypes[strings.ToLower(partition.Fstype)]; !typeAllowed {
+		if _, typeAllowed := fw.AllowedTypes[partition.Fstype]; !typeAllowed {
```
Comment (martinlindhe) [inline fs.go]: This was already correct too I think

File: fs.go (hunk 3 above)
Comment (troian) [reply fs.go]: Oops. To much cut

**Pull request merged**